sdk/documentintelligence/azure-ai-documentintelligence/samples/helper/utils.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
```

Comment:
@YalinLi0312 

Our PM got customer feedback that we should have more readable print result. That's why we introduced this utility file and share it among the samples. Let us know your thoughts on this. Thanks!

Reply:
Fixed it. Thanks.